app/react/Documents/components/scss/showToc.scss

```diff
@@ -0,0 +1,52 @@
+@import "../../../App/scss/config/_colors.scss";
```

Comment:
Why is this being imported? None of the color constants are being used in the file.

Reply:
@grafitto can you check this please ?

Reply:
Removed